sql/flow_config.go

```diff
@@ -79,3 +88,25 @@ func GetBaseDockerImageURI(configURL string) (string, error) {
 
 	return resp.BaseDockerImage, nil
 }
+
+func GetPythonSDKComptability(configURL, sqlCliVersion string) (astroRuntimeVersion, astroSDKPythonVersion string, err error) {
+	httpClient := &http.Client{}
```

Comment:
nit: this should go into the global var block, and be reused in all the methods of the package. There is no point in creating a new HTTP client for each individual method.